src/Process/$3dTilesProcessing.js

```diff
@@ -0,0 +1,174 @@
+    if (node.boundingVolume.region) {
+        // TODO
+    }
+    if (node.boundingVolume.box) {
+        return !camera.isBox3DVisible(node.boundingVolume.box, node.matrixWorld);
+    } else if (node.boundingVolume.sphere) {
+        // TODO return !camera.isSphereVisible(node.boundingVolume.sphere, node.matrixWorld);
+    }
+    return false;
+}
+
+let preSSE;
+export function pre3dTilesUpdate(context) {
+    // pre-sse
+    const hypotenuse = Math.sqrt(context.camera.width * context.camera.width + context.camera.height * context.camera.height);
+    const radAngle = context.camera.FOV * Math.PI / 180;
+
+     // TODO: not correct -> see new preSSE
+    // const HFOV = 2.0 * Math.atan(Math.tan(radAngle * 0.5) / context.camera.ratio);
+    const HYFOV = 2.0 * Math.atan(Math.tan(radAngle * 0.5) * hypotenuse / context.camera.width);
+    preSSE = hypotenuse * (2.0 * Math.tan(HYFOV * 0.5));
+}
+
+function computeNodeSSE(camera, node) {
+    if (node.boundingVolume.region) {
+        // TODO
+    }
+    if (node.boundingVolume.box) {
+        // TODO: compute proper distance
+        const worldCoordinateCenter = node.boundingVolume.box.getCenter();
+        worldCoordinateCenter.applyMatrix4(node.matrixWorld);
+        const distance = camera.camera3D.position.distanceTo(worldCoordinateCenter);
+        return preSSE * (node.geometricError / distance);
+    } else if (node.boundingVolume.sphere) {
+        const worldCoordinateCenter = node.boundingVolume.sphere.center;
+        worldCoordinateCenter.applyMatrix4(node.matrixWorld);
+        const distance = Math.max(
+            0.0,
+            camera.camera3D.position.distanceTo(worldCoordinateCenter) - node.boundingVolume.sphere.radius);
+        return preSSE * (node.geometricError / distance);
+    }
+    return Infinity;
+}
+
+export function init3dTilesLayer(context, layer) {
+    if (layer.initialised) {
+        return;
+    }
+    layer.initialised = true;
+
+    Fetcher.json(layer.url).then((tileset) => {
+        const urlPrefix = layer.url.slice(0, layer.url.lastIndexOf('/') + 1);
+        layer.tileIndex = new $3dTilesIndex(tileset, urlPrefix);
+        layer.asset = tileset.asset;
+        if (tileset.asset.gltfUpAxis === undefined || tileset.asset.gltfUpAxis === 'Y') {
+            layer.glTFRotation = (new THREE.Matrix4()).makeRotationX(Math.PI / 2);
+        } else if (tileset.asset.gltfUpAxis === 'X') {
+            layer.glTFRotation = (new THREE.Matrix4()).makeRotationZ(-Math.PI / 2);
+        } else if (tileset.asset.gltfUpAxis === 'Z') {
+            layer.glTFRotation = new THREE.Matrix4();
+        }
+        requestNewTile(context.view, context.scheduler, layer, tileset.root, undefined).then(
+            (tile) => {
+                context.view.scene.add(tile);
+                tile.updateMatrixWorld();
+                layer.root = tile;
+            });
+    });
+}
+
+export function process3dTilesNode(cullingTest, subdivisionTest) {
```

Comment:
The whole file looks a lot like `TiledNodeProcessing`.
How about using `TiledNodeProcessing` instead? Adapt it to cover both use-case and maybe rename it `TiledStructureProcessing` ?)